check50/_api.py

```diff
@@ -137,33 +137,33 @@ def import_checks(path):
     return mod
 
 
-def number_regex(number):
-    """
-    Create a regular expression to match the number exactly:
-        In case of a positive number:
-            (?<![\d\-])number(?!(\.?\d))
-        In case of a negative number:
-            number(?!(\.?\d))
-
-    (?<![\d\-]) = negative lookbehind, \
-        asserts that there are no digits and no - in front of the number.
-    (?!(\.?\d)) = negative lookahead, \
-        asserts that there are no digits and no additional . followed by digits after the number.
-
-    :param number: the number to match in the regex
-    :type number: any numbers.Number (such as int, float, ...)
-    :rtype: str
+class regex:
```

Comment:
I feel like this should just be a module rather than a class. A class that is just a collection of staticmethods is essentially the same as a module, but much less idiomatic.